x-pack/plugins/lens/public/xy_visualization/xy_suggestions.test.ts

```diff
@@ -111,18 +101,14 @@ describe('xy_suggestions', () => {
         },
         {
           isMultiRow: false,
-          columns: [strCol('foo'), numCol('bar')],
-          layerId: 'first',
-          changeType: 'unchanged',
-        },
-        {
-          isMultiRow: true,
-          columns: [unknownCol(), numCol('bar')],
+          columns: [numCol('bar')],
           layerId: 'first',
           changeType: 'unchanged',
         },
       ] as TableSuggestion[]).map((table) =>
-        expect(getSuggestions({ table, keptLayerIds: [] })).toEqual([])
+        expect(
+          getSuggestions({ table, keptLayerIds: [] }).every((suggestion) => suggestion.hide)
```

Comment:
This assertion will also pass if there are no suggestions. Please add another assertion that there is a specific length